src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +116,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(c -> new CandidateValidator().test(c))
```

Comment:
It is possible not to call the method 'test' itself, and simply pass the predicate
```suggestion
                .filter(new CandidateValidator())
```